src/App.js

```diff
@@ -1,12 +1,31 @@
 import React from "react"
 import './css/App.css';
-import Maps from "./Maps"
+import Callup from "./pages/callup/Callup"
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
+import NotFound from "./pages/404";
+import Menu from "./pages/menu/Menu";
+import OpenHouse from "./pages/open-house/OpenHouse";
+import GarageSale from "./pages/garage-sale/GarageSale";
+
 function App() {
   return (
     <div className="App">
-      {/*Map*/}
-      <Maps/>
+    <Router>
+      <Switch>
+        <Route  exact path="/" component={Menu} />
+        <Route  exact path="/callup" component={Callup} />
```

Comment:
I suggest that you push these calls to the menu component.

Reply:
I dont think so, it shouldn't matter, it is not rendering the components, it is just routing them :)